wl-clipboard-tools/Cargo.toml

```
@@ -0,0 +1,54 @@
+[package]
+name = "wl-clipboard-tools"
+version = "0.4.1" # remember to update html_root_url
+authors = ["Ivan Molodetskikh <yalterz@gmail.com>"]
+description = "Access to the Wayland clipboard for terminal and other window-less applications."
+edition = "2018"
+license = "MIT/Apache-2.0"
+
+readme = "README.md"
+documentation = "https://docs.rs/wl-clipboard-rs"
+repository = "https://github.com/YaLTeR/wl-clipboard-rs"
+keywords = ["wayland", "clipboard"]
+categories = ["command-line-utilities"]
+
+[badges]
+travis-ci = { repository = "YaLTeR/wl-clipboard-rs" }
+
+# [profile.dev]
+# panic = "abort"
```

```suggestion
version = "0.4.1"
```

I don't think all of this stuff is needed in a sub-crate.

---

edition was needed or else it wont compile.